datasetinsights/commands/download.py

```diff
@@ -79,8 +79,15 @@ def convert(self, value, param, ctx):
     help="Unity Simulation access token. "
     "This will override synthetic datasets source-uri for Unity Simulation",
 )
+@click.option(
+    "--checksum-file",
+    type=str,
+    default=None,
+    help="Dataset checksum text file. "
```

Comment:
Since `checksum-file` can either be a http url or file in the local filesystem, you should make this clear in the help message.